import_logs.py

```diff
@@ -1599,6 +1599,10 @@ def _get_hit_args(self, hit):
 
         args.update(hit.args)
 
+        # option --idsite was used, we need to force using this siteId
+        if site_id and site_id != args['idsite']:
+            args['idsite'] = site_id
```

Comment:
DynamicResolver will handle the case when replay tracking is used and when `--idsite` is passed, the StaticResolver instance is used (which returns just the idsite specified in the option), so I think instead of this code, it would be better to just remove `'idsite'` from `hit.args` before the `args.update(hit.args)`. Since idsite should be determined by the site resolver, and not `hit.args` (so it keeps the responsibility of handling --idsite to StaticResolver).